### PR TITLE
Final review and updates to meeting and faq pages

### PR DESCRIPTION
**Pull request by obigriffith**

### CODE REVIEW

File: docs/about/faq.rst [hunk 1]
```diff
@@ -20,15 +20,15 @@ Precision medicine refers to the customization of healthcare strategies to the i
...
+The rapid expansion of biomedical literature presents an intractable challenge for investigators and clinicians to efficiently interpret and synthesize the complete array of knowledge available for each cancer-associated genomic variant into clinically-relevant evidence statements. Efforts to curate the clinical importance of variants have been undertaken by a diverse, but fragmented, community of individuals. Crowdsourcing leverages the resources of individuals to produce a centralized, open-sourced platform for this diverse community to discuss and share findings in their respective specialities. CIViC also serves as a platform for contributors keep up to date on recent advances.
```
Comment (kkrysiak) [inline docs/about/faq.rst:23]:
```suggestion
The rapid expansion of biomedical literature presents an intractable challenge for investigators and clinicians to efficiently interpret and synthesize the complete array of knowledge available for each cancer-associated genomic variant into the underlying and synthesized clinically-relevant evidence. Efforts to curate the clinical importance of variants have been undertaken by a diverse, but fragmented, community of individuals. Crowdsourcing leverages the resources of individuals to produce a centralized, open-sourced platform for this diverse community to discuss and share findings in their respective specialities. CIViC also serves as a platform for contributors keep up to date on recent advances.
```

Comment (kkrysiak) [inline docs/about/faq.rst:23]: I don't know what was changed here. But given our evolution to assertions....should we be focused on clinically-relevant "evidence statements". Perhaps clinically-relevant utility? Just a thought.

File: docs/about/faq.rst [hunk 2]
```diff
@@ -20,15 +20,15 @@ Precision medicine refers to the customization of healthcare strategies to the i
...
+CIViC curatiors should express a strong interest in clinical cancer genomics. Curators should have expertise in relevant academic fields including basic cancer biology, oncology, genomics, clinical informatics, etc and ideally are training for or have already obtained a medical degree (e.g. PhD or MD). Additionally, we expect curators to be open to sharing information with the community, be respectful of other contributors, professional, truthful, ethical, and unbiased. There is sometimes pressure in academia to present one's research findings in a clinically relevant context. When CIViC evidence statements are created, the curator should conduct an objective assessment of how well the data supports any assertion of clinical relevance. All aspects of the CIViC project are open to debate and amendment, including these guidelines.
```
Comment (kkrysiak) [inline docs/about/faq.rst:31]: Should we note that Editors are required to fill out COIs here?

File: docs/about/faq.rst [hunk 3]
```diff
@@ -40,7 +40,7 @@ Variant descriptions will be accepted into CIViC provided there is evidence link
...
+An evidence statement is a brief description of the clinical relevance of a variant that has been determined by an experiment, trial, or study from a published literature source. It captures a variant's impact on clinical action, which can be predictive of therapy, correlated with prognostic outcome, inform disease diagnosis (i.e. cancer type or subtype), or predict predisposition to cancer in the first place. A single citation can be the source of multiple evidence statments, but each evidence statment has only one source. A single evidence statement should correspond to only one clinical interpretation and disease. For example, if a paper describes both predictive and prognostic relevance for a variant, two evidence statements should be created. If two publications draw the same conclusions about the clinical relevance of a variant, these should also be entered as two evidence statements.
```
Comment (kkrysiak) [inline docs/about/faq.rst:43]: We still need to consider functional here.

File: docs/about/faq.rst [hunk 4]
```diff
@@ -56,42 +56,42 @@ The :doc:`curation pages <../curating>` detail the curation and editing process,
...
 
-Why must my contributions be approved by a moderator?
------------------------------------------------------
-In an effort to ensure quality (and prevent automated spam), we require that edits be submitted to the review queue before they are shown as accepted. Moderators are used for this review step to protect the CIViC resource and will approve your revisions as soon as possible. An editor, moderator or administrator may comment on your proposed addition or revision. You will be able to see your new content in a pending state while it awaits review. To expedite the review process, we encourage you to submit high quality evidence (evidence level of validated or clinical) as the top priority. Similarly, proposing evidence statements for a new variant or gene may take longer for the community to review.
+Why must my contributions be approved by an editor?
```
Comment (kkrysiak) [inline docs/about/faq.rst:65]: Do we need to go formal case on this? Editor?

File: docs/about/faq.rst (hunk 1 above)
Comment (obigriffith) [reply docs/about/faq.rst]: Sure. I made another commit along these lines.

File: docs/about/faq.rst (hunk 2 above)
Comment (obigriffith) [reply docs/about/faq.rst:31]: Sounds good.

Comment (obigriffith) [reply docs/about/faq.rst]: added additional commit to address this

File: docs/about/faq.rst (hunk 4 above)
Comment (obigriffith) [reply docs/about/faq.rst:65]: Makes sense

File: docs/about/faq.rst (hunk 3 above)
Comment (obigriffith) [reply docs/about/faq.rst]: added something to this effect

**Comment by obigriffith** [comment]
@kkrysiak reviewed and requested changes. I made the requested changes. Rather than wait for re-review and approval I'm going to go ahead and merge.

**Pull request merged**